static/_redirects

```diff
@@ -370,7 +370,13 @@
 /overview/projects /learn/projects 301
 /subnets /learn/avalanche/subnets-overview 301
 
-#05/24/2023
+# 05/24/2023
 /apis/avalanchego/avalanchego-release-notes#v1912-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.9.12 301
 /apis/avalanchego/avalanchego-release-notes#v1714-view-on-github https://github.com/ava-labs/avalanchego/releases/tag/v1.7.14 301
 /apis/avalanchego/avalanchego-release-notes https://github.com/ava-labs/avalanchego/releases 301
+
+# 05/31/2023
+/subnets/customize-a-subnet.md#avalanchego-chain-configs docs/subnets/subnet-evm-avalanchego-config.md 301
```

Comment:
Please manually verify/correct each of these links. @meaghanfitzgerald can help you on this.